Canary-AWS-Bird-Automated-Deployment.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# This script automates the process for commissioning a bird that has been deployed as an AWS EC2 instance and configures the device personality using the Canary API. 
+# Values in the sample config.json file should be set. All bird services are disabled by default. 
+# Set the CANARY_HASH, CANARY_TOKEN, and FLOCK_ID variables based on your Canary console.
+# Sample code for automating the deployment of AWS EC2 birds can be found in the terraform folder of this repository
+
+# Retrieve flock info and set the FLOCK_ID variable to the desired flock if the default one won't be used
+curl https://$CANARY_HASH.canary.tools/api/v1/flocks/summary -d auth_token=$CANARY_TOKEN -G | jq '.flocks_summary[] | .flock_id' | sed 's/\"//g'
```

Comment:
Using the variable `CANARY_TOKEN` may be misleading here. Since we have Canarytokens as a separate meaning. Perhaps something like `AUTH_TOKEN`